kratos/utilities/convect_particles_utilities.h

```diff
@@ -112,7 +113,9 @@ template<std::size_t TDim> class ParticleConvectUtily
 
                 if(substep > 1 ) //first check if it falls within the same element
                 {
-                    is_found = mpSearchStructure->CalculatePosition(pelement->GetGeometry(), current_position[0], current_position[1], current_position[2], N);
+                    Geometry< Node < 3 > >& geom = pelement->GetGeometry();
+                    is_found = geom.IsInside(current_position, aux_point_local_coordinates, 1.0e-5);
```

Comment:
Actually, I have one comment:
Shouldn't this be a double::epsilon or whatever, 1.0e-5 seems a little bit magic

Reply:
This was the value already existing in the BinbasedPointLocator, so the best way to not break code it is to use the same value